deps/picotls/t/util.h

```diff
@@ -124,10 +124,27 @@ static inline void setup_session_file(ptls_context_t *ctx, ptls_handshake_proper
     }
 }
 
-static inline void setup_verify_certificate(ptls_context_t *ctx)
+static X509_STORE * openssl_init_cert_store(char const * crt_file)
+{
+    int ret = 0;
+    X509_STORE *store = X509_STORE_new();
+
+    if (store != NULL) {
```

Comment:
Do we want so silently fallback to using the default certificate store when `X509_STORE_new` return NULL? Assuming that that is _not_ the case, I think we should print a error message and exit.

Reply:
if input argument "crt_file" is NULL, default trust store should be used.

Reply:
updated. Thanks.

Reply:
It makes a lot of sense since this is a test utility. I've updated the change.